news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,124 @@
+
+## osu!catch
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/chronoxia.jpg)](https://osu.ppy.sh/s/429184)
+
+Our top pick is something that many osu!catch players would be familiar with due to it being selected for the osu!catch World Cup 2016's Finals mappool, and is now a fully ranked map. Both the Overdose made by Furely and the Chronos difficulties feature patterns that we don't usually see in most ranked maps - strict antiflow patterns, high overall spacing, and generally a lot of places to miss - while still making it a fun mapset of fair difficulty. Overall, the map is quite challenging for its star rating, being harder than other mapsets people usually see in the Ranked category. It also features several mappers, providing players with different styles across all difficulties. It's quite the engaging map, and one that could surely make you wiggle from joy!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/samurai.jpg)](https://osu.ppy.sh/s/594326)
+
+This mapset offers more of what we've come to expect from Ascendance with a well crafted series of difficulties that are a pleasure to play. All difficulties do a nice job of building up to the energetic kiai where the Evanesce difficulty stands out from the rest with its quick doubles that kick the difficulty up a notch. Overall while the mapset may not offer many surprises, its solid execution makes for a welcome addition to the catch spotlights.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/shelter.jpg)](https://osu.ppy.sh/s/536749)
+
+Despite of this set is being Myle's very first ranked set it came out to be a really enjoyable one. The spread built up to grew quite constantly. I really enjoyed playing it, he made a good work.
+
+## osu!mania
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/trigger.jpg)](https://osu.ppy.sh/s/555076)
+
+I have to say it. This map is definetly the most fun experience I had in a while from the ranked pool and I can only say thanks and congratulations to Wonki for winning this month spotlights and also for providing us such a good piece. The SVs on this map feels intense but they are not overdone and it just feels natural with the song, making it really confortable to play. Combine that with a cool usage of minijacks at the very end with some simple LN patterns this map just has everything you want to be the map of the month... and its art!
```

Comment:
First person, needs to spend less time waffling about winning the spotlight and more time explaining why. Better in the second half.